### ISSUE TITLE
gold-email-input accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/gold-email-input#7]
Audit results: 

[Severe] Controls and media elements should have labels (5)
[Warning] Text elements should have a reasonable contrast ratio (4)
Not applicable tests (9)

Contrast: hint text and "email" are too light. 

Windows / Firefox / NVDA: Again, text entry fields say "has autocomplete". Is this true?

**Comment by notwaldorf** [comment]
Done! :tada:
